src/main/java/com/epam/model/service/TextFileService.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.model.service;
+
+import com.epam.model.entity.TextInfoHolder;
+
+import java.io.*;
+import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.epam.model.entity.TextInfoHolder.Builder;
+
+public class TextFileService {
+    private final static String TEXT_FILE = "src/data.txt";
+    private final static String SENTENCE_COUNT_REGEX = "[.]\\s";
+    private final static String PARAGRAPH_COUNT_REGEX = "[\\s]{4}";
+    private static Builder builder;
+    private static StringBuilder text = new StringBuilder();
+    private static char[] symbols;
+    private static StringTokenizer words;
+
+    public static TextInfoHolder createInfoHolder() {
+        builder = new TextInfoHolder.Builder();
+        try {
+            text = readTextFile();
+            initSymbolArray();
+            initSymbolCount();
+            initWordCount();
+            findLongestWord();
+            initSentenceCount();
+            initParagraphCount();
+            return new TextInfoHolder(builder);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
```

Comment:
Remove sout. You need to create Exception and throw it to higher level. You can use sout only from view class.